spec/extensions/epic_interface_spec.rb

```diff
@@ -976,7 +976,7 @@ def strip_xml_whitespace!(root)
       'rpe' => 'urn:ihe:qrph:rpe:2009',
       'hl7' => 'urn:hl7-org:v3')
 
-      expect(node[1]).to be_equivalent_to(expected.root)
```

Comment:
This was also part of the 2.1.0 Study Type Question changes. Merging v3.0.0 should remove this